src/build/build.ts

```diff
@@ -26,6 +24,10 @@ import {PrefetchTransform} from './prefetch';
 import {waitFor} from './streams';
 import {parsePreCacheConfig} from './sw-precache';
 
+// TODO(fks) 09-22-2016: Latest npm type declaration resolves to a non-module
+// entity. Upgrade to import once compatible .d.ts file is released.
+import mergeStream = require('merge-stream');
```

Comment:
Interesting. I wonder if I'm gonna start seeing errors in other repos as I update to the real typescript v2.0.x